python/python/lance/indices.py

```diff
@@ -185,6 +197,60 @@ def train_ivf(
             )
             return IvfModel(centroids_array, distance_type)
 
+    def assign_ivf_partitions(
+        self,
+        ivf_model: IvfModel,
+        dst_dataset_uri: Optional[str],
```

Comment:
`dst_dataset_uri` => `output_uri`? we use less abbreviation in python SDKs.

Btw, should this be a dict arg, i.e., `output_uri = None` if we expect that user won't set it for most of the time?

Reply:
I can change to `output_uri`.  I could set it to `output_uri = None` (which would use a random tmp path?).  I think that would be confusing though since the other methods are going to need paths.  E.g. if a user is going down the "piece by piece" standalone route then they want to know and be aware of all the temp files because that is what they are trying to do.

Reply:
Ok, I came around.  I changed to `output_uri` and default to `None`.